Widgets/Animation/VarmintWidgetAnimationCollection.cs

```diff
@@ -32,6 +32,45 @@ public partial class VarmintWidgetAnimation
 
         //--------------------------------------------------------------------------------------
         /// <summary>
+        /// RotateConsistent - animate a rotation of a specified amount with a consistent speed
+        /// </summary>
+        //--------------------------------------------------------------------------------------
+        public static VarmintWidgetAnimation RotateLinear(
+            double durationSeconds,
+            float rotationAmount)
+        {
+            return new VarmintWidgetAnimation(durationSeconds, (widget, delta) =>
+            {
+                widget.Rotate = rotationAmount * (float)delta;
+            });
+        }
+
+        //--------------------------------------------------------------------------------------
+        /// <summary>
+        /// ScaleConsistent - animate a scale with a specified factor in relation to the current size with a consistent speed
+        /// </summary>
+        //--------------------------------------------------------------------------------------
+        public static VarmintWidgetAnimation ScaleLinear(
+            Vector2 originalSize,
+            double durationSeconds,
+            float scaleFactor)
+        {
+            return new VarmintWidgetAnimation(durationSeconds, (widget, delta) =>
+            {
+
+                // Animations scales from 1 (current size) to scale factor
+                float changeFactor = 0;
```

Comment:
This code works fine, however in general it is better if we can avoid using logic in favor of straight arithmetic.  Like this:

    var sizeChange = originalSize * scaleFactor - originalSize;
    ...
    widget.Size = originalSize + sizeChange * (float)delta;